io/zenoh-transport/tests/unicast_conduits.rs

```diff
@@ -346,6 +346,19 @@ fn conduits_tcp_only() {
     task::block_on(run(&endpoints));
 }
 
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.